diff --git a/files/gitlab-cookbooks/gitlab/recipes/gitlab-shell.rb b/files/gitlab-cookbooks/gitlab/recipes/gitlab-shell.rb
index 57f29c2..96ffe08 100644
--- a/files/gitlab-cookbooks/gitlab/recipes/gitlab-shell.rb
+++ b/files/gitlab-cookbooks/gitlab/recipes/gitlab-shell.rb
@@ -20,7 +20,7 @@ account_helper = AccountHelper.new(node)
 git_user = account_helper.gitlab_user
 git_group = account_helper.gitlab_group
 gitlab_shell_dir = "/opt/gitlab/embedded/service/gitlab-shell"
-gitlab_shell_var_dir = "/var/opt/gitlab/gitlab-shell"
+gitlab_shell_var_dir = "/opt/gitlab/var/gitlab-shell"
 git_data_directories = node['gitlab']['gitlab-shell']['git_data_directories']
 repositories_storages = node['gitlab']['gitlab-rails']['repositories_storages']
 ssh_dir = File.join(node['gitlab']['user']['home'], ".ssh")
