diff --git a/lib/gitlab/version.rb b/lib/gitlab/version.rb
index 3e095a9..3328f27 100644
--- a/lib/gitlab/version.rb
+++ b/lib/gitlab/version.rb
@@ -28,12 +28,12 @@ module Gitlab
         if @read_version.include?('-ee')
           "git@dev.gitlab.org:gitlab/gitlab-ee.git"
         else
-          "git@dev.gitlab.org:gitlab/gitlabhq.git"
+          "https://gitlab.com/gitlab-org/gitlab-ce.git"
         end
       when "GITLAB_SHELL_VERSION"
-        "git@dev.gitlab.org:gitlab/gitlab-shell.git"
+        "https://gitlab.com/gitlab-org/gitlab-shell.git"
       when "GITLAB_WORKHORSE_VERSION"
-        "git@dev.gitlab.org:gitlab/gitlab-workhorse.git"
+        "https://gitlab.com/gitlab-org/gitlab-workhorse.git"
       when "GITLAB_PAGES_VERSION"
         "https://gitlab.com/gitlab-org/gitlab-pages.git"
       else
diff --git a/support/set-revisions b/support/set-revisions
index e9b5bf4..a6cb1ba 100755
--- a/support/set-revisions
+++ b/support/set-revisions
@@ -8,10 +8,10 @@ main() {
   if [[ "$gitlab_rails_ref" == *"-ee"* ]] ; then
     gitlab_rails_repo=git@dev.gitlab.org:gitlab/gitlab-ee.git
   else
-    gitlab_rails_repo=git@dev.gitlab.org:gitlab/gitlabhq.git
+    gitlab_rails_repo=https://gitlab.com/gitlab-org/gitlab-ce.git
   fi
-  gitlab_shell_repo=git@dev.gitlab.org:gitlab/gitlab-shell.git
-  gitlab_workhorse_repo=git@dev.gitlab.org:gitlab/gitlab-workhorse.git
+  gitlab_shell_repo=https://gitlab.com/gitlab-org/gitlab-shell.git
+  gitlab_workhorse_repo=https://gitlab.com/gitlab-org/gitlab-workhorse.git
 
   gitlab_rails_ref_id=$(check_remote_ref gitlab_rails)
   assert_non_empty gitlab_rails_ref_id
